src/components/forms/Checkbox/Checkbox.tsx

```diff
@@ -20,6 +21,7 @@ export const Checkbox = ({
   inputRef,
   tile,
   labelDescription,
+  indeterminate,
```

Comment:
This works the same as UWSDS' implementation though I'll note that I expected to be able to go from indeterminate --click--> checked --click--> unchecked. Instead it goes back to indeterminate. 

Since that's how the USWDS implementation works, nothing to change here!